snooty.toml

```diff
@@ -25,12 +26,14 @@ sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 driver-short = "PyMongo"
 driver-long = "PyMongo, the MongoDB synchronous Python driver,"
 driver-async = "PyMongo Async"
+django-odm = "MongoDB Backend for Django"
```

Comment:
`django-odm` == `django-mongodb-backend` ?

Reply:
Yes! The name of the source constant isn't visible - it renders as "MongoDB Backend for Django" and we just use it for writing simplicity/consistency.